index.bs

```diff
@@ -602,6 +613,16 @@ Modify the [=discard a document|discard=] algorithm for {{Document}}s by appendi
   1. [=list/Empty=] <var ignore>document</var>'s [=Document/post-prerendering activation steps list=].
 </div>
 
+<h2 id="interaction-with-other-specs">Interaction with other specifications and concepts</h2>
+
+<h3 id="interaction-with-visibility-state">Interaction with Page Visibility</h3>
+
+Documents in [=prerendering browsing contexts=] are [=Document/hidden=].
```

Comment:
Note that for portals this might not be true, and portals are envisioned to be a type of prerendering browsing context. We might want to state that subtlety here as a TODO or Note, even though portals spec work is kind of on hold while we get these foundations worked out...

Reply:
Done, added it as an Issue which seemed consistent with the other Issues about portals.